Tests/ClientTests/Messaging/GleanPlumbMessageTests.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+@testable import Client
+
+class GleanPlumbMessageTests: XCTestCase {
+    // MARK: - Properties
+    var subject: GleanPlumbMessage!
+
+    let experimentKey = "not-a-silly-experiment-"
+    let messageID = "not-a-silly-experiment-en"
+
+    // MARK: - Setup & Teardown
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        subject = nil
+        super.tearDown()
+    }
+
+    // MARK: - Test isUnderExperimentWith
+    func testMessageIsUnderExperiment_BadExperimentKey() {
+        subject = createMessage(messageId: messageID)
+
+        XCTAssertFalse(subject.isUnderExperimentWith(key: nil))
+    }
+
+    func testMessageIsUnderExperiment_isControlIsTrue() {
```

Comment:
nit: consistency 
```suggestion
    func testMessageIsUnderExperiment_isControl_isTrue() {
```

Reply:
Hah! Technically, this is the condition "`isControl` is true", and not the expectation of the test being true. I can see the confusion. I'll rename appropriately.